src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -239,7 +245,8 @@ main.help.opt.any=\
 \ Operation modifiers valid in any mode:\n\
 \n\
 \  -C DIR                     Change to the specified directory and include the\n\
-\                             following file
+\                             following file. When used in extract mode, extracts\n\
+\                             the jar to the specified directory
```

Comment:
Should this be updated on line 187 as well in the compatibility mode section?

Reply:
Updated in latest version of the PR.